paper.tex

```diff
@@ -446,15 +447,15 @@ \section{Strings}\label{sec:strings}
     \item \deff{contains}: checks whether it contains $x$
```

Comment:
@EugeneDar Why not `contained`?

Reply:
@Graur I think name `contains` indicates that the method returns the bool type. Let me try to explain my point of view. If you were to explain what the method does, it would sound something like "check if string `contains` ...". It seems more appropriate to me to use `contained` in case the method would return a new string modified by some rule. As an example I would give the `text.trimmed` method, here we return the passed string, but after a certain change. Do you know what I mean, or do I need to be clearer.